dist/images/ovnkube.sh

```diff
@@ -917,6 +923,7 @@ ovn-master() {
     --logfile-maxage=${ovnkube_logfile_maxage} \
     ${hybrid_overlay_flags} \
     ${disable_snat_multiple_gws_flag} \
+    ${disable_pkt_mtu_check_flag} \
```

Comment:
does this flag need to be passed to the `ovnkube-master`?

Reply:
no I suppose not, I just copied it everywhere. Will remove.